go.mod

```diff
@@ -137,6 +146,7 @@ require (
 	github.com/prometheus/procfs v0.7.3 // indirect
 	github.com/rivo/uniseg v0.2.0 // indirect
 	github.com/ryanuber/go-glob v1.0.0 // indirect
+	github.com/shirou/gopsutil/v3 v3.22.4
```

Comment:
The changes to the dependencies come from this library I use to check if the agent process is running. It is better to pull in this dependency than check manually in my opinion due to difference in operating systems.

There may come a point where we run the agent on start-up and we won't need to worry about if it is running. When we get there we can remove this library.